src/Slices/ServiceInstanceDetails/Test/Page.test.tsx

```diff
@@ -411,6 +411,125 @@ describe("ServiceInstanceDetailsPage", () => {
       ),
     ).not.toBeVisible();
 
+    // Events
```

Comment:
Wouldn't that be better to have the tabs split, just in case if something break then get better overview what

Reply:
It would slow down the tests considerably to reinstantiate an entire page for each tab, so that's why I did it like this